### PR TITLE
proposals#edit form text in track field is added

### LINKED ISSUE #1659: proposals#edit form show text in track field
The field for track selection should show some text, like difficulty level does.

### PR DESCRIPTION
**Pull request by namangupta01**
closes #1659

### CODE REVIEW

File: app/views/proposals/_proposal_form.html.haml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
     = speaker_selector_input f
 
-    = track_selector_input f
+    = f.input :track, as: :select, collection: @conference.program.tracks, input_html: { class: 'select-help-toggle' }, include_blank: '(Please categorize your event)' if @conference.program.tracks.any?
```
Comment (differentreality) [inline app/views/proposals/_proposal_form.html.haml]: Can you please explain this change here?
Why are changing the code for the collection of the available tracks?

Also one line below, the equivalent text for difficulty level is '(Please select)'. We should not say the same thing to our users in different ways.

Comment (namangupta01) [reply app/views/proposals/_proposal_form.html.haml]: @differentreality I got it and fixed it! Please have a look!

**Comment by differentreality** [comment]
Great @namangupta01 
Please add screenshots of your changes. 
Also please avoid squashing your commits in open PRs, instead add new commits until review is over ;)
Please check the failing linters, and fix accordingly so that we can merge your changes!

**Comment by namangupta01** [comment]
Here are the screenshots :
screenshot from 2017-09-13 16-58-05

screenshot from 2017-09-13 16-57-49

**Comment by namangupta01** [comment]
@differentreality as my travis build is passed. Should i squash my commits?

**Comment by differentreality** [comment]
Very nice @namangupta01 :+1: 
Yes go ahead and squash your commits!

**Comment by namangupta01** [comment]
@differentreality done! :+1:

**Comment by differentreality** [comment]
Thanks @namangupta01, nicely done :+1:

**Pull request merged**